js/self-starting-sequence/index.ts

```diff
@@ -0,0 +1,19 @@
+import { ReadableApp } from "@scramjet/types";
+import { defer } from "@scramjet/utility";
+
+const app: ReadableApp<string> = async function(_stream,interval:number) {
+    const hub = this.hub!;
```

Comment:
pls check if this.hub is optional. probably you can remove ! now

Reply:
![image](https://user-images.githubusercontent.com/53794300/231395298-5e0add0d-ef13-4008-90a2-c4e2e436bc14.png)

Still having the same issue